jobbergate-cli/pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[tool.poetry]
+name = "jobbergate-cli"
+version = "1.1.1"
+description = "Jobbergate CLI Client"
+authors = ["Omnivector Solutions <info@omnivector.solutions>"]
+license = "MIT"
+packages = [ { include = "jobbergate_cli" } ]
+
+
+[tool.poetry.dependencies]
+python = "^3.6.2"
+click = "^8.0.1"
+inquirer = "^2.7.0"
+PyJWT = "<2"
+PyYAML = "^5.4.1"
+python-dotenv = "^0.19.0"
+requests = "^2.26.0"
+tabulate = "^0.8.9"
+urllib3 = "^1.26.7"
+sentry-sdk = "^1.4.3"
+boto3 = "^1.18.64"
+loguru = "^0.5.3"
+
+[tool.poetry.dev-dependencies]
+black = "^21.9b0"
+pytest-cov = "^3.0.0"
+pytest-random-order = "^1.0.4"
+pytest-freezegun = "^0.4.2"
+pytest-responsemock = "^1.0.1"
+flake8 = "^3.9.2"
+isort = "^5.9.3"
+pytest-env = "^0.6.2"
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "--random-order --cov=jobbergate_cli --cov-report=term-missing --cov-fail-under=25"
+env = [
+    "JOBBERGATE_API_ENDPOINT = https://jobbergate-api-staging.omnivector.solutions",
+    "JOBBERGATE_DEBUG = false",
+]
+testpaths = ["jobbergate_cli/test"]
+
+[tool.black]
+
```

Comment:
shouldn't `line-length` be set with value 120?

Reply:
There was some discussion about this at one point. Folks were going back and forth between 100 and 120, so I split the difference. It's consistent across black, flake8, and isort in this project.

Reply:
I just mean because flake8 is set to check 120 but the default is 110. I'm just used to keep it the same

Reply:
In the latest commit, I standardized the max line-length to 110 on both jobbergate-api and jobbergate-cli